readme.md

```diff
@@ -17,13 +17,35 @@
 <span><img width='250px' src='./preview/print_E.png'/></span>
 <span><img width='250px' src='./preview/print_F.png'/></span>
 
-## Git LFS
-This project uses Git [Large File Support](https://git-lfs.github.com), so only pointers to the files are handled by Git. You must install LFS to work with the actual files when using Git.
+## LFS and file download
+If you only want to download the Blender and STL files **DO NOT USE download ZIP** GitHub button, since it is not compatible with LFS (Large File Storage), but instead get the files from the [latest release](https://github.com/inputlabs/alpakka_case/releases/latest) package.
 
-## Download all files (.blender, .stl, .3mf)
-If you want to download all files **DO NOT USE GitHub download ZIP** option, since it is not compatible with LFS, but instead get the files from the [latest released version](https://github.com/inputlabs/alpakka_case/releases/latest) package.
+To use Git with this project it is required to install Git [Large File Storage](https://git-lfs.github.com).
 
-## Recommended print settings
-It is very recommended to use the **predefined layer heights** to achieve good results (with acceptable tolerances).
 
-Check the [Manual](https://inputlabs.io/devices/alpakka/manual/diy_case) or use the included `.3mf` files for details.
+## Repeated and mirrored parts
+Some parts require to print 2x or 4x units, and some of them are mirrored. For simplicity, and to avoid mistakes, we export the STL files with these repetitions and mirroring baked in.
+
+If you want to split these to arrange them better in the printer bed:
+
+**In PrusaSlicer**
+- Right click in object.
+- Split > To objects.
+
+**In Cura**
+- Open Marketplace.
+- Install `Mesh Tools` plugin.
+- Right click in object.
+- Mesh Tools > Split model into parts.
+
+## File labels
```

Comment:
The mixture of model(s) in the .stl and (partial) settings (for a specific printer / model / type) in the filename is IMO very bad.
It would instead be preferable to use the filename / path only / primarily as identifier and refer all settings elsewhere, such as the documentation.